packages/windapsearch/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=windapsearch
+pkgver=25.3679b37
+pkgrel=1
+pkgdesc='Script to enumerate users, groups and computers from a Windows domain through LDAP queries.'
+arch=('x86_64')
+groups=('blackarch' 'blackarch-windows')
```

Comment:
Add `blackarch-automation  blackarch-recon  blackarch-scanner` groups and remove `blackarch-windows`, the windows group is for .exe tools that usually needs to run in Windows systems.